src/main/java/org/hackedtogether/xpadditions/data/ModRecipeProvider.java

```diff
@@ -0,0 +1,36 @@
+package org.hackedtogether.xpadditions.data;
+
+import net.minecraft.data.*;
```

Comment:
Don't import all of a package

Reply:
All of the package is used, so intellij automatically does this